uaclient/cli.py

```diff
@@ -1579,6 +1631,37 @@ def action_refresh(args, *, cfg: config.UAConfig):
     return 0
 
 
+def configure_apt_proxy(
+    cfg: config.UAConfig, scope: AptProxyScope, args: Dict
+) -> None:
+    """
+    Handles setting part the apt proxies - global and uaclient scoped proxies
+    """
+    # needs to import from within here or else unit tests fail
+    # due to wrong importation path issues i.e
+    # tests will use cli.setup_apt_proxy over apt.setup_apt_proxy
+
+    set_value = args.get("set_value")
+    set_key = args.get("set_key")
```

Comment:
Mypy complains that this value cannot be `None`, because you are using the `in` operator below
```suggestion
    set_value = args.get("set_value", "")
    set_key = args.get("set_key", "")
```

Reply:
Yeah, good solution.